istack-commons/buildtools/pom.xml

```diff
@@ -22,7 +22,7 @@
     </parent>
 
     <artifactId>istack-commons-buildtools</artifactId>
-    <name>istack-commons-buildtools</name>
+    <name>istack common utility buildtools</name>
```

Comment:
Wouldn't be nicer to name it with capital letter there. To be fair I am not sure if istack project is even named with capital letter.

Reply:
I made it consistent with the rest - main pom has the name with upper case, modules with lower case